xbmc/pictures/GUIWindowPictures.cpp

```diff
@@ -502,6 +502,8 @@ void CGUIWindowPictures::GetContextButtons(int itemNumber, CContextButtons &butt
   CGUIMediaWindow::GetContextButtons(itemNumber, buttons);
   if (item && !item->GetProperty("pluginreplacecontextitems").asBoolean())
     buttons.Add(CONTEXT_BUTTON_SETTINGS, 5);                  // Settings
+
+  BaseContextMenuManager::Get().AppendVisibleContextItems(item, buttons);
```

Comment:
would some sort of "priority" parameter make sense? Some addons might add an entry for better usability that should be listet on top (like a delete button some forum users demand to be on first level and not below "manage")